frontend/packages/knative-plugin/src/components/overview/RoutesOverviewListItem.tsx

```diff
@@ -12,19 +12,36 @@ export type RoutesOverviewListItemProps = {
 const RoutesOverviewListItem: React.FC<RoutesOverviewListItemProps> = ({ route, resource }) => {
   const {
     metadata: { name, namespace },
-    status: { url },
+    status: {
+      url,
+      traffic: trafficData = [{ revisionName: resource.metadata.name, url: route.status.url }],
+    },
   } = route;
-  const trafficData = _.find(route.status.traffic, {
-    revisionName: resource.metadata.name,
+  const filterTrafficBasedOnResource = (tr: Traffic) =>
+    referenceFor(resource) === referenceForModel(ServiceModel) ||
+    (referenceFor(resource) === referenceForModel(RevisionModel) &&
+      tr.revisionName === resource.metadata.name);
+
+  return trafficData.filter(filterTrafficBasedOnResource).map((traffic: Traffic) => {
+    const routeUrl = traffic?.url || url;
+    const trafficPercent = traffic.percent ? `${traffic.percent}%` : null;
+    return (
+      <li className="list-group-item">
+        <div className="row">
+          <div className="col-sm-10 col-xs-10">
```

Comment:
nit: as we have `col-xs-10` so is `col-sm-10` needed

Reply:
fixed